express/assets/javascripts/cookies.js

```diff
@@ -60,32 +61,44 @@ var cookieBanner = function () {
         });
     }
 
-    function getCookieDomain () {
-        const PROD_HOSTNAME = 'sign-in.service.gov.uk';
+    function isProdDomain() {
```

Comment:
❤️  I'm a big fan of this sort of thing. The less thinking I have to do, the better!